boards/stm32f429i-disco/Kconfig

```diff
@@ -13,20 +13,8 @@ config BOARD_STM32F429I_DISCO
     default y
     select CPU_MODEL_STM32F429ZI
 
-    # Put defined MCU peripherals here (in alphabetical order)
-    select HAS_PERIPH_DMA
-    select HAS_PERIPH_I2C
-    select HAS_PERIPH_SPI
-    select HAS_PERIPH_TIMER
-    select HAS_PERIPH_UART
-    select HAS_PERIPH_USBDEV
-
     # Put other features for this board (in alphabetical order)
     select HAS_HIGHLEVEL_STDIO
-    select HAS_RIOTBOOT
-
-    # Clock configuration
-    select BOARD_HAS_HSE
-    select BOARD_HAS_LSE
 
 source "$(RIOTBOARD)/common/stm32/Kconfig"
+source "$(RIOTBOARD)/stm32f429i-disc1/Kconfig"
```

Comment:
Note that this is not the normal way we do it, usually this would be factored into a common board or something instead of just taking another board...  This is what is done in the makefile and TBH, we should probably deprecate this board as it is super obsolete.

Reply:
See #17376

Reply:
@aabadie do you have the old or new version?